### ISSUE TITLE
Draw text in Meshcat viewer

### ISSUE THREAD

**Issue by Victorlouisdg** [airo-ugent/airo-drake#1]
I'd like to draw text in meshcat in two ways:
1. As a title banner to keep track what the viewer is for e.g "Animation of robot moving home", similar to the "No connection" banner
2. Text in the 3D view e.g. next to a frames "pregrasp pose" and "grasp pose" etc.  

![image](https://github.com/airo-ugent/airo-drake/assets/23340946/6fd28b15-151e-4a8d-88b5-e88665f39b33)

**Comment by adverley** [comment]
Absolutely. I don't think such functionality is exposed but maybe we can send some packets to the server from python. See for example [here](https://github.com/meshcat-dev/meshcat-python): "Your code can use the meshcat python libraries or communicate directly with the server through its [ZeroMQ](http://zguide.zeromq.org/) socket."
